### DISCUSSION TITLE
Can't load winpe iso

### DISCUSSION THREAD

**Discussion by DenisVASI9** [ipxe/ipxe#283] [Q&A] [answered]
Hi!
I created win pe in windows using deployment tools and want to boot it via http.
I use this nginx config:
server {
        listen       80;
        location ~ \.(iso) {
            root /home/devbook/iso;
    	}
}
And this ipxe config:

#!ipxe
dhcp

cpuid --ext 29 && set arch x64 || set arch x86
cpuid --ext 29 && set archl amd64 || set archl i386

:loop
sanhook --drive 0x81 ${root-path}
sanboot --keep --no-describe http://192.168.88.99/winpe.iso || goto loop

At the end I get an error - Could not open SAN device

**Comment by mcb30** [comment, marked as answer]
@DenisVASI9 Don't try to use `sanboot` to boot a `.iso` image.  There are many online resources available explaining why this is a bad idea.

Instead, use https://ipxe.org/wimboot to download and boot your WinPE files (that you have already created) directly via HTTP.

**Reply by DenisVASI9** [reply]
I'm having a little trouble understanding in a new competency for me, please tell me why this is a bad idea.

Thank you very much!

**Reply by mcb30** [reply]
Basically: every operating system installer packaged as an `.iso` will expect to find the installer files on an attached real DVD-ROM drive.  This doesn't work for an iSCSI boot, since the virtual DVD-ROM necessarily disappears as soon as the operating system kernel starts up.
